### ISSUE TITLE
there are two "晚上" when I select a time in Chinese language

### ISSUE THREAD

**Issue by hanfengcan** [metabase/metabase#11014]
**Describe the bug**
when I select a time in Chinese language, there are two "晚上". It is AM or PM in English language.

I found that the reason is `moment.localeData()`.

when I try to fix this, the language is English. 

how can I change the English to Chinese when i am developing

**Screenshots**
![image](https://user-images.githubusercontent.com/4559753/65844669-bdfc2380-e369-11e9-8663-1847fae01b01.png)

**Comment by hanfengcan** [comment]
when I try to use 'lein ring server'. It is broke.

log
```
{:clojure.main/message "Syntax error (IOException) compiling at (E:\\temp\\form-init142022581749813005.clj:1:77).\r\nServer returned HTTP response code: 500 for URL: http://localhost:3000/\r\n",
 :clojure.main/triage
 {:clojure.error/phase :compile-syntax-check,
  :clojure.error/line 1,
  :clojure.error/column 77,
  :clojure.error/source "form-init142022581749813005.clj",
  :clojure.error/path "E:\\temp\\form-init142022581749813005.clj",
  :clojure.error/class java.io.IOException,
  :clojure.error/cause "Server returned HTTP response code: 500 for URL: http://localhost:3000/"},
 :clojure.main/trace
 {:via
  [{:type clojure.lang.Compiler$CompilerException,
    :message "Syntax error compiling at (E:\\temp\\form-init142022581749813005.clj:1:77).",
    :data {:clojure.error/phase :compile-syntax-check, :clojure.error/line 1, :clojure.error/column 77, :clojure.error/source "E:\\temp\\form-init142022581749813005.clj"},
    :at [clojure.lang.Compiler load "Compiler.java" 7648]}
   {:type java.io.IOException,
    :message "Server returned HTTP response code: 500 for URL: http://localhost:3000/",
    :at [sun.reflect.NativeConstructorAccessorImpl newInstance0 "NativeConstructorAccessorImpl.java" -2]}
   {:type java.io.IOException,
    :message "Server returned HTTP response code: 500 for URL: http://localhost:3000/",
    :at [sun.net.www.protocol.http.HttpURLConnection getInputStream0 "HttpURLConnection.java" 1876]}],
  :trace
  [[sun.net.www.protocol.http.HttpURLConnection getInputStream0 "HttpURLConnection.java" 1876]
   [sun.net.www.protocol.http.HttpURLConnection getInputStream "HttpURLConnection.java" 1474]
   [java.net.HttpURLConnection getResponseCode "HttpURLConnection.java" 480]
   [javax.swing.JEditorPane getStream "JEditorPane.java" 733]
   [javax.swing.JEditorPane setPage "JEditorPane.java" 432]
   [javax.swing.JEditorPane setPage "JEditorPane.java" 880]
   [javax.swing.JEditorPane <init> "JEditorPane.java" 274]
   [sun.reflect.NativeConstructorAccessorImpl newInstance0 "NativeConstructorAccessorImpl.java" -2]
   [sun.reflect.NativeConstructorAccessorImpl newInstance "NativeConstructorAccessorImpl.java" 62]
   [sun.reflect.DelegatingConstructorAccessorImpl newInstance "DelegatingConstructorAccessorImpl.java" 45]
   [java.lang.reflect.Constructor newInstance "Constructor.java" 423]
   [clojure.lang.Reflector invokeConstructor "Reflector.java" 305]
   [clojure.java.browse_ui$open_url_in_swing invokeStatic "browse_ui.clj" 15]
   [clojure.java.browse_ui$open_url_in_swing invoke "browse_ui.clj" 15]
   [clojure.lang.Var invoke "Var.java" 384]
   [clojure.java.browse$open_url_in_swing invokeStatic "browse.clj" 64]
   [clojure.java.browse$browse_url invokeStatic "browse.clj" 74]
   [clojure.java.browse$browse_url invoke "browse.clj" 66]
   [ring.server.standalone$open_browser_to invokeStatic "standalone.clj" 38]
   [ring.server.standalone$open_browser_to invoke "standalone.clj" 37]
   [ring.server.standalone$serve$fn__4247 invoke "standalone.clj" 107]
   [ring.server.standalone$try_port invokeStatic "standalone.clj" 16]
   [ring.server.standalone$try_port invoke "standalone.clj" 10]
   [ring.server.standalone$serve invokeStatic "standalone.clj" 100]
   [ring.server.standalone$serve doInvoke "standalone.clj" 75]
   [clojure.lang.RestFn invoke "RestFn.java" 423]
   [ring.server.leiningen$serve invokeStatic "leiningen.clj" 19]
   [ring.server.leiningen$serve invoke "leiningen.clj" 16]
   [user$eval76095 invokeStatic "form-init142022581749813005.clj" 1]
   [user$eval76095 invoke "form-init142022581749813005.clj" 1]
   [clojure.lang.Compiler eval "Compiler.java" 7177]
   [clojure.lang.Compiler eval "Compiler.java" 7167]
   [clojure.lang.Compiler load "Compiler.java" 7636]
   [clojure.lang.Compiler loadFile "Compiler.java" 7574]
   [clojure.main$load_script invokeStatic "main.clj" 475]
   [clojure.main$init_opt invokeStatic "main.clj" 477]
   [clojure.main$init_opt invoke "main.clj" 477]
   [clojure.main$initialize invokeStatic "main.clj" 508]
   [clojure.main$null_opt invokeStatic "main.clj" 542]
   [clojure.main$null_opt invoke "main.clj" 539]
   [clojure.main$main invokeStatic "main.clj" 664]
   [clojure.main$main doInvoke "main.clj" 616]
   [clojure.lang.RestFn applyTo "RestFn.java" 137]
   [clojure.lang.Var applyTo "Var.java" 705]
   [clojure.main main "main.java" 40]],
  :cause "Server returned HTTP response code: 500 for URL: http://localhost:3000/",
  :phase :compile-syntax-check}}
```

**Comment by hanfengcan** [comment]
final, I use `lein run` and `yarn build ` to run the metabase

**Comment by flamber** [comment]
Hi @hanfengcan 
Please use the forum for questions and troubleshooting: https://discourse.metabase.com/
As for the `晚上` on both `AM` and `PM`, that's a translation problem. You can help improving the translation by going to https://poeditor.com/join/project/ynjQmwSsGh
If you just want to fix it on your development build, then translations are located in `./locales/`.
And please close this issue.

**Comment by hanfengcan** [comment]
hey, @flamber 
It is not a problem of translation. the AM/PM is from the 3rd party libraries `moment`.

**Comment by flamber** [comment]
@hanfengcan Then I'm unsure if I understand the problem. Along with questions about `lein`.
Is it because `晚上` means `AM` *and* `PM`, but it's shown twice?
Could you explain the problem with more details?

**Comment by hanfengcan** [comment]
maybe try this code
```
moment.localeData('en').meridiem(0) // AM
moment.localeData('en').meridiem(12) // PM

// in Chinese
moment.localeData('zh-cn').meridiem(0) // 晚上
moment.localeData('zh-cn').meridiem(12) // 晚上

// but
moment.localeData('zh-cn').meridiem(0, 0) // 凌晨
moment.localeData('zh-cn').meridiem(12, 0) // 中午
```

I found the problem is in the hoursMinutesInput.jsx.
 `{moment.localeData().meridiem(12)}` and `{moment.localeData().meridiem(0)}`

sorry, my English is poor

**Comment by paulrosenzweig** [comment]
Fixed in #11049
